test/common/http/conn_manager_impl_test.cc

```diff
@@ -588,8 +590,10 @@ TEST_F(HttpConnectionManagerImplTest, PathFailedtoSanitize) {
 
   EXPECT_CALL(*filter, encodeHeaders(_, true));
   EXPECT_CALL(response_encoder_, encodeHeaders(_, true))
-      .WillOnce(Invoke([](const HeaderMap& headers, bool) -> void {
+      .WillOnce(Invoke([filter](const HeaderMap& headers, bool) -> void {
```

Comment:
minor nit: should we be consisting in how we pass the filter to the lamba? its implicitly passed by & elsewhere